admin/schedule-form-excludes.php

```diff
@@ -298,12 +298,15 @@
 							$size = $site_size->filesize( $file );
 
 							if ( false !== $size ) :
-								$size          = $size ?: '0';
-								$excluded_size = $excluded_site_size->filesize( $file ) ?: '0'; ?>
+								$size          = $size ?: 0;
+								$excluded_size = $excluded_site_size->filesize( $file ) ?: 0;
```

Comment:
@willmot how does this look? Are you happy with `0 B of x KB` or shall I look into removing format after `0`?
<img width="1242" alt="screen shot 2016-07-07 at 14 20 10" src="https://cloud.githubusercontent.com/assets/2185972/16654494/f9c96e90-444d-11e6-921e-9171208f4f22.png">